records_mover/db/postgres/copy_options/csv.py

```diff
@@ -95,6 +95,7 @@ def postgres_copy_options_csv(unhandled_hints: Set[str],
             cant_handle_hint(fail_if_cant_handle_hint, 'quoting', hints)
     else:
         _assert_never(mode)
+    quiet_remove(unhandled_hints, 'quoting')
```

Comment:
It looks like we weren't clearing this out in this path, which one of the now-fixed unit tests revealed.  Not sure why this wasn't seen before!